src/pages/transfer/transfer.ts

```diff
@@ -14,7 +22,137 @@ import { IonicPage, NavController, NavParams } from "ionic-angular";
   templateUrl: "transfer.html",
 })
 export class TransferPage {
```

Comment:
The indentation in this class seems off. Is everything indented with 2 spaces instead of 4?